### PR TITLE
Os master squashed

### PR DESCRIPTION
**Pull request by fabianmurariu**
### What changes were proposed in this pull request?
break out the raphtory-arrow crate

### Does this PR introduce any user-facing change? If yes is this documented?
No
### How was this patch tested?
Unit

### CODE REVIEW

File: .github/workflows/manual_release_rust.yml [hunk 1]
```diff
@@ -15,4 +15,7 @@ jobs:
   call-release-github-workflow:
     name: _Release 4 - Publish to Github
     uses: ./.github/workflows/_release_github.yml
+    with:
```
Comment (Haaroon) [inline .github/workflows/manual_release_rust.yml:18]: change to inputs

File: resource/netflowsorted/v2_sorted/v2.parquet
Comment (Haaroon) [inline resource/netflowsorted/v2_sorted/v2.parquet:1]: we need to upload the parquet files? are you sure?

**Review by Haaroon — COMMENTED** [review]
added some comments

**Review by miratepuffin — APPROVED** [review]
Not finished going through yet 

One thought is we need some tests which show everything passes if you don't have access to diskbrown

**Pull request merged**